llvm/test/CodeGen/AArch64/trampoline.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: llc -mtriple=aarch64-- < %s | FileCheck %s
```

Comment:
`-mtriple=aarch64` specifies a generic ELF triple (yes, ELF is the default:) )

`; UNSUPPORTED: darwin, system-windows` should not be needed.

Reply:
OK, I'll remove that.